### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#467] [Esami & Esoneri M-Z]
### Traccia

<img width="745" alt="image" src="https://github.com/user-attachments/assets/30094ba8-ffb4-4527-96eb-b40769c14b4e">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/4ff51eeb-ba38-4a14-8424-a767a6908fb3)
